microsoft-edge/devtools-guide-chromium/speed/get-started.md

```diff
@@ -22,440 +22,480 @@ ms.date: 05/04/2021
+Use the **Performance** tool to analyze what work the main thread is doing while the page loads, and find ways to defer or remove unnecessary work:
 
-1. Expand the **Bottom-Up** section.  This tab breaks down what activities took up the most time.  If nothing is displayed in the **Bottom-Up** section, click the label for **Main** section.  The **Bottom-Up** section only shows information for whatever activity, or group of activity, you have currently selected.  For example, if you chose one of the `mineBitcoin` activities, the **Bottom-Up** section is only going to show information for that one activity.
+1. Open the **Performance** tool.
 
-   ![The Bottom-Up tab](../media/speed-glitch-tony-remix-performance-slow-network-slow-cpu-timings-summary-minebitcoin.msft.png)
+1. In the **Performance** tool, click **Start profiling and reload the page** (![Refresh](../media/reload-icon.msft.png)), and then click **Stop** once the page has fully loaded.
 
-The **Self Time** column shows you how much time was spent directly in each activity.  For example, in the following figure, about 63% of main thread time was spent on the `mineBitcoin` function.
+    DevTools displays a visualization of all the work that the browser performed in order to load the page. This visualization is referred to as the **trace**:
 
-It's time to see whether using production mode and reducing JavaScript activity may speed up the page load.  Start with production mode:
+    ![The Performance tool trace of the page load](./images/performance-trace.png)
 
-1. In the editor tab, open `webpack.config.js`.
+The trace shows activity chronologically, from left to right. The CPU and NET charts at the top give you an overview of CPU and network activity. The light brown area in the CPU chart corresponds to a period of time when the browser was busy with scripting activity. This area is a clue that you may be able to speed up page load by doing less JavaScript work.
 
-1. Change `"mode":"development"` to `"mode":"production"`.
+![A long scripting activity in the CPU chart](./images/perf-busy-script.png)
 
-1. Wait for the new build to deploy.
+Investigate the trace to find ways to do less JavaScript work.
 
-1. Audit the page again.
+The **Main** section shows a chronological log of main thread activity, from left to right.  The y-axis (top to bottom) shows why events occurred. For example, in the following figure, the browser spends most of the time in a **Parse HTML** event. As long as this event runs, the webpage isn't entirely displayed. This event caused a long **Evaluate Script** event, which corresponds to when the browser executes JavaScript code. The **Evaluate Script** event caused an `(anonymous)` function to run, which caused the `initRatings` function to run:
```

Comment:
```suggestion
The **Main** section shows a chronological log of main thread activity, from left to right.  The y-axis (top to bottom) shows why events occurred. For example, in the following figure, the browser spends most of the time in a **Parse HTML** event. As long as this event runs, the webpage isn't entirely displayed. This event caused a long **Evaluate Script** event, which corresponds to when the browser executes JavaScript code.

The **Evaluate Script** event caused an `(anonymous)` function to run, which caused the `initRatings` function to run:
```
arbitrarily break up too-long para. to deliver as series of 2- or 3-sentence paragraphs, separately readable/ility.

Reply:
Thanks. I think I'll break it earlier, after the "y-axis" sentence. It feels more logical to me. Let me know what you think.